### PR TITLE
Made snackbar component

### PR DESCRIPTION
**Pull request by poojavishnoi**

### CODE REVIEW

File: components/Snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+.snackbar_container a {
+  color: var(--primary-text-color);
+  text-decoration: none;
+  font-size: 1.2rem;
+  padding: 0.5rem 0.7rem;
+}
+
+.snackbar_container a:hover {
+  background-color: var(--primary-text-color);
+  color: white;
```
Comment (AtulHande03a) [inline components/Snackbar/snackbar.css]: use variables for color

**Review by AtulHande03a — APPROVED** [review]
review: use variables for colors , rest all looks good and naming also looks good

File: components/Snackbar/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+.baseline_snackbar,
+.stacked_snackbar {
+  left: 50%;
+  transform: translate(-50%, 0px);
```
Comment (anandkumarcoder) [inline components/Snackbar/snackbar.css]: translateX can be used instead of translate

File: components/Snackbar/snackbar.css [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+.btn {
+  padding: 0.7rem;
+  background-color: white;
+  margin: 1rem;
+  border: none;
+  cursor: pointer;
+  background-color: var(--primary-color);
+  color: white;
```
Comment (anandkumarcoder) [inline components/Snackbar/snackbar.css]: #FFFFFF can be used

File: components/Snackbar/snackbar.css [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  border-radius: 0.5rem;
+}
+
+.btn:hover {
+  background-color: var(--primary-text-color);
+}
+
+.snackbar_button {
+  margin-left: 10%;
```
Comment (anandkumarcoder) [inline components/Snackbar/snackbar.css]: you have used rem everywhere else

File: components/Snackbar/snackbar.css [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+  .stacked_snackbar {
+    width: 16rem;
+  }
+
+  .snackbar_button a {
+    font-size: 0.4rem;
+  }
+
+  .snackbar_button a:nth-child(1) {
+    width: 3rem;
```
Comment (anandkumarcoder) [inline components/Snackbar/snackbar.css:127]: line 97 has %. matter of preference choice. try to use consistent values

File: components/Snackbar/snackbar.css [hunk 6]
```diff
@@ -0,0 +1,128 @@
+
+:root {
+  --primary-color: #ff7474;
+  --primary-text-color: #ff5454;
+  --background-color: #fde3e3;
+  --border-color: #d3d3d3;
+}
```
Comment (anandkumarcoder) [inline components/Snackbar/snackbar.css:8]: better to keep all variable items in one CSS file

**Review by anandkumarcoder — APPROVED** [review]
looks good apart from small changes

File: components/Snackbar/snackbar.css (hunk 1 above)
Comment (poojavishnoi) [reply components/Snackbar/snackbar.css]: done

File: components/Snackbar/snackbar.css (hunk 2 above)
Comment (poojavishnoi) [reply components/Snackbar/snackbar.css]: Okay.Great suggestion

File: components/Snackbar/snackbar.css (hunk 3 above)
Comment (poojavishnoi) [reply components/Snackbar/snackbar.css]: done

File: components/Snackbar/snackbar.css (hunk 4 above)
Comment (poojavishnoi) [reply components/Snackbar/snackbar.css]: done

File: components/Snackbar/snackbar.css (hunk 5 above)
Comment (poojavishnoi) [reply components/Snackbar/snackbar.css:127]: okay.

File: components/Snackbar/snackbar.css (hunk 6 above)
Comment (poojavishnoi) [reply components/Snackbar/snackbar.css:8]: okay

**Pull request merged**